src/main/java/Commands/InvalidCommand.java

```diff
@@ -0,0 +1,17 @@
+package Commands;
+
+import Tasks.TaskList;
+import util.Storage;
+import util.Ui;
+
+public class InvalidCommand extends DukeCommand {
```

Comment:
Personally, I think if a `Command` is invalid, it should not be considered as a `Command`, therefore no inheritance. A better implementation to me is probably to remove this particular `InvalidCommand`, and directly throw exceptions when parsing fails.